boards/arm/rpi_pico/rpi_pico.dts

```diff
@@ -68,3 +68,17 @@
 &gpio0 {
 	status = "okay";
 };
+
+&i2c0 {
+	clock-frequency = <100000>;
+	status = "okay";
+	pinctrl-0 = <&i2c0_default>;
+	pinctrl-names = "default";
+};
+
+&i2c1 {
+	clock-frequency = <100000>;
```

Comment:
```suggestion
	clock-frequency = <I2C_BITRATE_STANDARD>;
```

include/dt-bindings/i2c/i2c.h

Reply:
I forgot about this macro, now it is fixed.